content/en/docs/components/pipelines/pipelines-quickstart.md

```diff
@@ -2,74 +2,38 @@
 title = "Pipelines Quickstart"
 description = "Getting started with Kubeflow Pipelines"
 weight = 10
-                    
-+++
-{{% alert title="Out of date" color="warning" %}}
-This guide contains outdated information pertaining to Kubeflow 1.0. This guide
-needs to be updated for Kubeflow 1.1.
-{{% /alert %}}
 
++++                 
 {{% stable-status %}}
 
-Use this guide if you want to get a simple pipeline running quickly in
-Kubeflow Pipelines.
-
-* This quickstart guide shows you how to use one of the samples that come with 
-  the Kubeflow Pipelines installation and are visible on the Kubeflow Pipelines
-  user interface (UI). You can use this guide as an introduction to the 
-  Kubeflow Pipelines UI.
-* The end-to-end tutorial shows you how to prepare and compile a pipeline, 
-  upload it to Kubeflow Pipelines, then run it.
-
-## Deploy Kubeflow and open the pipelines UI
-
-Follow these steps to deploy Kubeflow and open the pipelines dashboard:
+Use this guide if you want to get an introduction to the Kubeflow Piplines user interface (UI) and get a simple pipeline running quickly. 
 
-1. Follow the guide to [deploying Kubeflow on GCP](/docs/gke/deploy/).
+The goal with this quickstart guide is to shows how to use two of the samples that come with 
+the Kubeflow Pipelines installation and are visible on the Kubeflow Pipelines
+UI. You can use this guide as an introduction to the 
+Kubeflow Pipelines UI.
 
-    {{% pipelines-compatibility %}} 
+## Deploy Kubeflow and open the Kubeflow Pipelines UI
 
-1. When Kubeflow is running, access the Kubeflow UI at a URL of the form
-  `https://<deployment-name>.endpoints.<project>.cloud.goog/`, as described in the setup
-  guide. The Kubeflow UI looks like this:
-  <img src="/docs/images/central-ui.png" 
-    alt="Kubeflow UI"
-    class="mt-3 mb-3 border border-info rounded">
+There are several options to [deploy Kubeflow Pipelines](/docs/components/pipelines/installation/), follow the option that best suits your needs. If you are uncertain and just want to try out kubeflow pipelines it is recommended to start with the [standalone deployment](/docs/components/pipelines/installation/standalone-deployment/).
```

Comment:
Nit: I would recommend linking directly to the installation options page

[deploy Kubeflow Pipelines](/docs/components/pipelines/installation/overview/)